solidity/script/avs/DeployAVS.s.sol

```diff
@@ -226,4 +227,55 @@ contract DeployAVS is Script {
             require(strategies[i] != address(0), "Strategy address is 0");
         }
     }
+
+    function addREZStrategy() external {
+        IERC20 rezToken = IERC20(0x3B50805453023a91a8bf641e279401a0b23FA6F9);
+        IStrategyFactory strategyFactory = IStrategyFactory(
+            0x5e4C39Ad7A3E881585e383dB9827EB4811f6F647
+        );
+
+        IStrategy rezStrategy = strategyFactory.deployedStrategies(rezToken);
+
+        ECDSAStakeRegistry stakeRegistry = ECDSAStakeRegistry(
+            0x272CF0BB70D3B4f79414E0823B426d2EaFd48910
+        );
+
+        // get current quorum
+        Quorum memory currentQuorum = stakeRegistry.quorum();
+        Quorum memory newQuorum;
+        newQuorum.strategies = new StrategyParams[](
+            currentQuorum.strategies.length + 1
+        );
+
+        uint256 totalStrategies = newQuorum.strategies.length;
+        uint96 baseMultiplier = uint96(10000 / totalStrategies);
+
+        uint96 remainder = 10000 % uint96(totalStrategies);
+
+        for (uint256 i = 0; i < currentQuorum.strategies.length; i++) {
+            newQuorum.strategies[i] = currentQuorum.strategies[i];
+            newQuorum.strategies[i].multiplier = baseMultiplier;
+        }
```

Comment:
do we really want to apply a prorate quorum for REZ?

Reply:
Doesn't matter, we'll read it manually for ISM config.